### PR TITLE
Doc for using agent to consume hydra powered APIs.

### PR DESCRIPTION
**Pull request by priyanshunayan**
This PR adds: 
* a short introduction of Agent
* Instructions on how to use Agent package
* Instructions on how to use Agent GUI

### CODE REVIEW

File: pages/hydra-agent-redis-graph.md [hunk 1]
```diff
@@ -4,6 +4,235 @@ title: Redis as a graph database for Hydra Agent | Hydraecosystem.org
 permalink: /hydra-agent-redis-graph
 ---
 
+# Introduction:
+
+The hydra-python-agent is a smart Hydra client implemented in Python which works with[ Hydrus](https://github.com/HTTP-APIs/hydrus). Smart clients are generic automated clients that establish resilient connected data networks leveraging knowledge graphs.
```
Comment (xadahiya) [inline pages/hydra-agent-redis-graph.md:9]: The agent works with all hydra based servers not just hydrus

Comment (priyanshunayan) [reply pages/hydra-agent-redis-graph.md]: Right! I was so used to using Hydrus that it came out naturally. I have fixed it now.

**Comment by priyanshunayan** [comment]
@xadahiya Thank you for reviewing. I have made the changes.

File: pages/hydra-agent-redis-graph.md (hunk 1 above)
Comment (priyanshunayan) [reply pages/hydra-agent-redis-graph.md]: @xadahiya ^^

**Pull request merged**